### PR TITLE
Support Scala 2.13

### PR DESCRIPTION
**Pull request by dhs3000**
I updated the dependencies and ran tests locally (successful).

We are evaluating whether to use Scaldi and for that it would be good if future versions of Scala are supported.

### CODE REVIEW

File: src/main/scala-2.13/scaldi/Wire.scala [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+          paramLists.flatten find (_.isLeft) match {
+            case Some(Left(errorMsg)) =>
+              error(c, errorMsg)
+            case _ =>
+              val wiredType = paramLists.foldLeft (Select(New(Ident(tpe.typeSymbol)), termNames.CONSTRUCTOR): c.Tree) {
+                case (acc, l) => Apply(acc, l collect {case Right(v) => v})
+              }
+
+//              c.info(c.enclosingPosition, "Wired: " + c.universe.show(wiredType), false)
```
Comment (dave-handy) [inline src/main/scala-2.13/scaldi/Wire.scala]: probably should be removed or changed to trace... having trouble finding the type of `c`

**Comment by dave-handy** [comment]
Looks great to me... unfortunately I don't know if anyone is around with write access. Thanks for putting this together!

**Comment by otrosien** [comment]
any news on this?

**Comment by dave-handy** [comment]
Unfortunately no... see https://github.com/sangria-graphql/sangria/issues/446

**Pull request closed without merging**